frontend/src/__tests__/cypress/cypress/pages/acceleratorProfile.ts

```diff
@@ -0,0 +1,240 @@
+import { Modal } from './components/Modal';
+import { TableRow } from './components/table';
+
+class AcceleratorRow extends TableRow {
+  findDescription() {
+    return cy.findByTestId('description');
+  }
+}
+
+class TolerationRow extends TableRow {
+  shouldHaveOperator(name: string) {
+    this.find().find(`[data-label=Operator]`).should('have.text', name);
+    return this;
+  }
+
+  shouldHaveEffect(name: string) {
+    this.find().find(`[data-label=Effect]`).should('have.text', name);
+    return this;
+  }
+
+  shouldHaveTolerationSeconds(toleration: string) {
+    this.find().find(`[data-label="Toleration Seconds"]`).should('have.text', toleration);
+    return this;
+  }
+}
+
+class AcceleratorProfile {
+  visit() {
+    cy.visitWithLogin('/acceleratorProfiles');
+    this.wait();
+  }
+
+  sortedByColumn(column: string, order: string) {
+    return this.findColumn(column).then(($cells) => {
+      const cellValues = Array.from($cells).map((cell) => cell.textContent?.trim());
+      const sortedCellValues =
+        order === 'asc'
+          ? [...cellValues].sort()
+          : [...cellValues].sort((a, b) =>
+              b === undefined || a === undefined ? -1 : b.localeCompare(a),
+            );
```

Comment:
I suppose this works here, however note that sorting is not always based purely on the visible text.

Reply:
should i change the function?

Reply:
It's probably fine for now. 

Since sorting, filtering, etc, is all table related and similar across the app, we'll likely need a story to make generic tools for testing so that each page isn't doing their own thing. It can be refactored at that point.